queries/javascript/highlights.scm

```diff
@@ -180,6 +180,7 @@
   "%="
 ] @operator
 
+(binary_expression "/" @operator)
```

Comment:
Can this just move up into the list of operators above it?

Reply:
I do not know if this would be 100% accurate. In the spec its its own delineation from the other punctuation's. So I wanted it to match the case stated. I can move it to simplify but I am not sure.